src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooBuildPlanService.java

```diff
@@ -173,6 +173,16 @@ private Stage createBuildStage(ProgrammingLanguage programmingLanguage, final bo
                 tasks.add(0, checkoutTask);
                 return defaultStage.jobs(defaultJob.tasks(tasks.toArray(new Task[0])).finalTasks(testParserTask));
             }
+            case C -> {
+                // Do not run the builds in extra docker containers if the dev-profile is active
+                if (!activeProfiles.contains(JHipsterConstants.SPRING_PROFILE_DEVELOPMENT)) {
```

Comment:
now we have some duplicated code. Can we create a new method: getDockerImage(String programmingLanuage), then the duplication would not be necessary